level1a/handlers/level1a.py

```diff
@@ -22,11 +22,17 @@
 OFFSET_FACTOR = 2
 ORBIT_FREQUENCY = 0.1
 ATTITUDE_FREQUENCY = 1.
+HTR_FREQUENCY = 0.1
 
 PLATFORM_PREFIXES = {
     "HK_ecPowOps_1", "PreciseAttitudeEstimation", "PreciseOrbitEstimation",
     "scoCurrentScMode", "TM_acGnssOps", "TM_afAcsHiRateAttitudeData",
 }
+HTR_COLUMNS = [
+    "TMHeaderTime", "HTR1A", "HTR1B", "HTR1OD",
+    "HTR2A", "HTR2B", "HTR2OD", "HTR7A", "HTR7B", "HTR7OD", "HTR8A",
+    "HTR8B", "HTR8OD",
```

Comment:
```suggestion
    "TMHeaderTime", "HTR1A", "HTR1B", "HTR1OD",
    "HTR2A", "HTR2B", "HTR2OD", "HTR7A", "HTR7B", "HTR7OD",
    "HTR8A", "HTR8B", "HTR8OD",
```

Reply:
Maybe? Either something like that or strict 80 alignment (the latter probably works best with auto-formatting).

Reply:
Fixeddd